helm-xref.el

```diff
@@ -182,11 +185,13 @@ Needs to be set the value of `xref-show-xrefs-function'."
                                (cons (cons 'fetched-xrefs xrefs)
                                      alist))))))
 
-(if (< emacs-major-version 27)
-    (setq xref-show-xrefs-function 'helm-xref-show-xrefs)
-  (progn
-    (setq xref-show-xrefs-function 'helm-xref-show-xrefs-27)
-    (setq xref-show-definitions-function 'helm-xref-show-defs-27)))
+;;;###autoload
+(progn
```

Comment:
Are you sure this works here ? I've never seen an autoload for other stuff than functions.

Reply:
I tested by doing this yesterday:

``` elisp
(use-package helm
  :config
  (helm-mode 1))

(use-package helm-xref
  :defer t)
```

and using `xref-find-definitions` on something that had two choices for the definition. I noticed `helm` being used, so I think the `autoload` did its job there.